boards/arc/axs/support/openocd.cfg

```diff
@@ -0,0 +1,80 @@
+#  Copyright (C) 2019 Synopsys, Inc.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+# Configure JTAG cable
+# SDP has built-in FT2232 chip, which is similar to Digilent HS-1, except that
+# it uses channgel B for JTAG, instead of channel A.
+interface ftdi
+
+# Only specify FTDI serial number if it is specified via
+# "set _ZEPHYR_BOARD_SERIAL 12345" before reading this script
+if { [info exists _ZEPHYR_BOARD_SERIAL] } {
+       ftdi_serial $_ZEPHYR_BOARD_SERIAL
+}
+
+ftdi_vid_pid 0x0403 0x6010
+ftdi_layout_init 0x0088 0x008b
+ftdi_channel 1
+
+
+adapter_khz 10000
+
+# ARCs supports only JTAG.
+transport select jtag
+
+
+#
+# HS Development Kit SoC.
+#
+# Contains dual-core ARC HSxx.
+#
+
+source [find cpu/arc/hs.tcl]
+
+set _coreid 0
+set _dbgbase [expr 0x00000000 | ($_coreid << 13)]
+
+# CHIPNAME will be used to choose core family (600, 700 or EM). As far as
+# OpenOCD is concerned EM and HS are identical.
+set _CHIPNAME arc-em
+
+# OpenOCD discovers JTAG TAPs in reverse order.
+
+set _TARGETNAME2 $_CHIPNAME.cpu2
+jtag newtap $_CHIPNAME cpu2 -irlen 4 -ircapture 0x1 -expected-id 0x200424b1
+set _TARGETNAME1 $_CHIPNAME.cpu1
+jtag newtap $_CHIPNAME cpu1 -irlen 4 -ircapture 0x1 -expected-id 0x200024b1
+
+################################
```

Comment:
I would suggest to move core 0 definition to the end so that it gets selected by OpenOCD by default, see https://github.com/zephyrproject-rtos/zephyr/commit/7a8c624f53338d52876c120ff1a7afbcd36bd3b9 for more details.